### DISCUSSION TITLE
Sorry, How To Show Mouse Cursor After Hiding Mouse Cursor?

### DISCUSSION THREAD

**Discussion by ghost** [nesbox/TIC-80#2085] [Q&A]
Hi,

If we do:
```
function TIC()
 poke(0x3FFB,0)
```
we can hide the mouse cursor.

How do we show the mouse cursor after hiding it?
Let me know, thanks!

Jesse

**Comment by nesbox** [comment]
As described in the wiki
https://github.com/nesbox/TIC-80/wiki/RAM#mouse-cursor
pls call `poke(0x3FFB,128)`
